openml/setups/sklearn_converter.py

```diff
@@ -0,0 +1,33 @@
+from openml.flows import OpenMLFlow
+from openml.setups import OpenMLParameter
+
+
+def openml_param_name_to_sklearn(openml_parameter, flow):
```

Comment:
In my opinion, this function should be moved to flow/sklearn_converter submodule to have the scikit-learn code as close together as possible.

Reply:
Yes, maybe. Given that we have the ambition to abstract away from this is the (not so) close future, it makes sense to keep this in one place.